pkg/handler3/handler.go

```diff
@@ -213,6 +215,9 @@ func (o *OpenAPIService) HandleGroupVersion(w http.ResponseWriter, r *http.Reque
 			if clause.SubType != accepts.SubType && clause.SubType != "*" {
 				continue
 			}
+			if accepts.SubType == subTypeProtobufDeprecated {
+				fmt.Println("Deprecated: use SubType com.github.proto-openapi.spec.v3.v1.0+protobuf instead.")
+			}
```

Comment:
We were mainly trying to fix https://github.com/kubernetes/kube-openapi/pull/333/files#r1063082961 with this issue. Let's add back the Content-Type header return

Reply:
Sorry I didn't get I didn't remove any content-type headers.

Reply:
We should add in L206-207 from that link into this PR to show that Content-Type doesn't return an error anymore

Reply:
We should still not set the content-type for deprecate subtype right

Reply:
Since we're returning actual content even if the client passes in the deprecated subtype, I think it does it make sense to return the content-type? The content-type may need to be fixed up so none of the special symbols are contained in the return.

Reply:
We should set the content-type to the right, new content-type both for the deprecated and the new accept value.